web/src/main/java/no/nav/modiapersonoversikt/rest/persondata/PersondataService.kt

```diff
@@ -127,6 +127,20 @@ class PersondataServiceImpl(
         )
     }
 
+    fun erGyldigGT(gt: String?): Boolean {
+        return if (gt != null) {
+            when (gt) {
+                "0301" -> false
+                "4601" -> false
+                "5001" -> false
+                "1103" -> false
+                else -> true
+            }
+        } else {
+            false
+        }
+    }
```

Comment:
Kan kanskje forenkles til noe alà;
```suggestion
    val GT_SOM_BURDE_HA_BYDEL = listOf("0301", "4601", "5001", "1103")
    fun erGyldigGT(gt: String?): Boolean {
        return when {
          gt == null -> false
          GT_SOM_BURDE_HA_BYDEL.contains(gt) -> false
          else -> true
        }
    }
```